app/templates/cli.js

```diff
@@ -1,9 +1,13 @@
 #!/usr/bin/env node
-'use strict';
+<% if (babel) { %>
+import meow from 'meow';
+import <%= camelModuleName %> from './';
+
+const cli = meow([<% } else { %>'use strict';
```

Comment:
template strings can be used on node 4 also, so I though that it could be updated in other PR.